packages/api-headless-cms/src/content/plugins/index.ts

```diff
@@ -55,6 +55,17 @@ export default (
                 return;
             }
 
+            if (
```

Comment:
What happens when a user doesn't have a full-access but does have the required scopes?

Reply:
@Fsalker I don't really get it; why do you even need to check the `id` and `access` fields? The idea is that if `context.user !== null`, then the user provided a JWT or a PAT and is authenticated. Which at this point is all we need. That should work for both `read` and `manage`.  

`context.user` is only set when: there is a valid JWT/PAT. That's it. 

At this point we're only checking if a user is `authenticated`. We don't care about authorization until we solve those other issues we talked about.